pallets/delegation/src/migrations/v1.rs

```diff
@@ -16,9 +16,18 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
-use crate::*;
-
-use frame_support::{storage::bounded_btree_set::BoundedBTreeSet, IterableStorageMap, StorageMap, StoragePrefixedMap};
+use crate::{
+	deprecated::{
+		v1::storage::{Children, Delegations, Roots},
+		v2::{storage::DelegationNodes, DelegationNode as DelegationNodeV2},
+	},
+	migrations::DelegationStorageVersion,
+	Config, DelegationDetails, DelegationHierarchies, DelegationHierarchyDetails, DelegationNodeIdOf, Permissions,
+	StorageVersion, Weight,
+};
+use frame_support::{
```

Comment:
Did you spent time on making the old migrations work?
Parity deletes old migrations after a while. Maybe we should start doing the same. We cannot support old migrations infinity anyways

Reply:
Yeah it took me quite some time because of just importing `use crate::*` which caused a bug without a helping error message. I would be super happy to just get rid of it.